style.css

```diff
@@ -405,9 +437,189 @@ nav {
+@media only screen and (min-width: 768px) {
   body {
-    width: 1600px;
-    margin: auto;
+    width: 100%;
+  }
+
+  header {
+    width: 100%;
+  }
+
+  header nav {
+    display: flex;
+    justify-content: space-between;
+    width: 90%;
+    margin: 0 auto;
+  }
+
+  header nav .hamburger {
+    display: none;
+  }
+
+  .menu-items {
+    display: flex;
+    list-style: none;
+  }
+
+  .menu-items li {
+    margin-right: 32px;
+  }
+
+  .menu-items li a {
+    text-decoration: none;
+    color: var(--paragraph);
+    font-size: 15px;
+    font-weight: 500;
+  }
+
+  .about-section,
+  .contact-section,
+  .contact-form {
+    width: 100%;
+  }
+
+  .about-section {
+    margin-top: 47px;
+    background-image: url(images/bg-desktop1.svg);
+    align-items: center;
+  }
+
+  .primary-text,
+  .social-media {
+    width: 100%;
+  }
+
+  .about-section .primary-text {
+    width: 80%;
+    margin: 0 auto;
+  }
+
+  .about-section .social-media {
+    margin-bottom: 430px;
+  }
+
+  .works-section {
+    margin-top: 142px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .work-card,
+  .work-reverse {
+    width: 90%;
+    align-items: center;
+    justify-content: space-around;
+    height: 496px;
+  }
+
+  .work-card {
+    flex-direction: row;
+  }
+
+  .work-reverse {
+    flex-direction: row-reverse;
+  }
+
+  .work-image {
+    display: none;
+  }
+
+  .work-image-large {
+    display: flex;
+    margin: 1%;
+  }
+
+  .work-image-large img {
+    width: 100%;
+  }
+
+  .work-details {
+    width: 50%;
+  }
+
+  .work-details h2 {
+    width: 80%;
+  }
+
+  .work-description {
+    width: 90%;
+  }
+
+  .button-margin {
+    width: 50%;
+    padding-left: 5%;
+  }
+
+  .contact-section {
+    flex-direction: row;
+    padding: 10%;
+  }
+
+  .contact-left {
+    width: 50%;
+    margin-top: -8%;
+  }
+
+  .experience {
+    width: 50%;
+  }
+
+  .contact-left .primary-margin-change {
+    width: 90%;
+    text-align: left;
+  }
+
+  .contact-left .resume-btn {
+    width: 35%;
+  }
+
+  .experience .inner-list {
+    flex-direction: row;
+    align-content: center;
+    justify-content: space-evenly;
+  }
+
+  .experience .inner-list li {
+    margin-top: 36px;
+    width: 30%;
+    padding: 12px 12px 20px 12px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .experience .inner-list li span {
+    padding-top: 16px;
+    margin-left: -1%;
+  }
+
+  .contact-form {
+    margin-top: -10%;
+    background-image: url(images/bg-desktop2.svg);
+  }
+
+  .contact-form p {
+    width: 569px;
+  }
+
+  .contact-list-item input {
+    width: 447px;
+    border-radius: 0;
+    margin-bottom: 27px;
+  }
+
+  .contact-list-item textarea {
+    width: 447px;
+    border-radius: 0;
+  }
+
+  .contact-list .contact-list-item button:enabled {
+    width: 30%;
+    margin-left: 35%;
+    margin-top: 20px;
+    font-size: 17px;
+    font-weight: 500;
   }
 }

```

Comment:
- [ ] The responsiveness of the site breaks at some screen sizes. I will suggest you take into consideration removing the widths that you set for specific sections just as mentioned by the previous reviewer's [comment](https://github.com/normainobary2021/my-portfolio/pull/5#discussion_r916217297). Setting the widths will cause responsive issues. 

Please a gif of the demo I ran on your site. Please do take note of the areas where I moved my mouse cursor frequently.  They are the sections that break out and some do overlap. Like the languages in the about section.

Demo |
:----------------:|
![a1](https://user-images.githubusercontent.com/39523312/178048519-900bf908-ac84-40cf-b081-d83e6e68bdad.gif)